Fixed #83

MattJeanes opened the pull request:
While issue of client props not being made sometimes persists (seems like a GMod issue), I've suppressed the errors with IsValid checks.

weltensturm commented on WAC Aircraft/lua/entities/wac_hc_base/cl_init.lua:
```diff
@@ -177,9 +179,13 @@ function ENT:OnRemove()
+			t.model:Remove()
+		end
+	end
+	if IsValid(t.model) then
```
are you sure about this one?

MattJeanes commented:
I made an oops. Just fixed it though. It's because I was messing around with all sorts of other code trying to get it to work, but then I reverted everything and re-did the checks and accidentally missed that out.
